BeerOverflowWindowsApp/BarComparers/ComparerByDistance.cs

```diff
@@ -21,11 +21,11 @@ public int Compare(BarData barData1, BarData barData2)
             }
             else if (distance1 < distance2)
             {
```

Comment:
Inverted the values so it worked as I wanted. Looking at it now, it's a hacky solution :/

Reply:
I think this is fine. The whole point of iComparer is to sort it the way we need it to.